bandcamp_dl/__main__.py

```diff
@@ -86,7 +86,6 @@ def main():
     logger = logging.getLogger(logging_handle)
 
     # TODO: Its possible to break bandcamp-dl temporarily by simply erasing a line in the config, catch this and warn.
-    arguments = config.init_config(parser)
```

Comment:
init_config() was removed by https://github.com/iheanyi/bandcamp-dl/pull/229 and bandcamp-dl would no longer run for me unless I removed that line.

Reply:
Looks like the merge for that PR did _not_ go properly considering its still there, hopefully nothing else slipped through.

Reply:
Yup, and my change resolved https://github.com/iheanyi/bandcamp-dl/issues/232 for me.  No idea if something else is broken because of the changes the other PR made though.